Modix.Common.Test/InvalidLengthResultTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Modix.Common.ErrorHandling;
+using NUnit.Framework;
+using Shouldly;
+
+namespace Modix.Common.Test
+{
+    public class InvalidLengthResultTests
+    {
+        [Test]
```

Comment:
`TestCase` bro.

Reply:
Refactored - hope I did that right.

Reply:
I've never actually used the `ExpectedResult` property of `TestCase` before. Kinda forgot it was there.

I would just have made it another parameter `bool isSuccess` to the test method itself, and done `result.IsSuccess.ShouldBe(isSuccess)`. This would provide the very very very minor benefit of having the failure message be "result.IsSuccess should have been X but was Y" as opposed to... whatever message NUnit generates that won't have "result.IsSuccess" in it.